setup.cfg

```diff
@@ -71,13 +71,16 @@ docs =
   jplephem
 
 
```

Comment:
```suggestion
```

Reply:
Whitespace is important for readability.

Reply:
Fixed.